test/fuzz/RecoveryController/TransferLogic.fuzz.t.sol

```diff
@@ -0,0 +1,56 @@
+/**
+ * Created by Pragma Labs
+ * SPDX-License-Identifier: MIT
+ */
+pragma solidity 0.8.19;
+
+import {RecoveryController_Fuzz_Test} from "../RecoveryController.fuzz.t.sol";
+
+/**
+ * @notice Fuzz tests for the transfer logic of "RecoveryController".
+ */
+contract TransferLogic_Fuzz_Test is RecoveryController_Fuzz_Test {
+    /* ///////////////////////////////////////////////////////////////
+                              SETUP
+    /////////////////////////////////////////////////////////////// */
+
+    function setUp() public override {
+        RecoveryController_Fuzz_Test.setUp();
+    }
+
+    /*//////////////////////////////////////////////////////////////
+                            TRANSFER LOGIC
+    //////////////////////////////////////////////////////////////*/
+
+    function testFuzz_Revert_transfer(address user, address to, uint256 initialBalance, uint256 amount) public {
```

Comment:
Should we also overwrite & block the approve & permit function? No real effect though on the logic

Reply:
Don't see the reason why if transferFrom is blocked?